#!/usr/bin/env python
PACKAGE = "laser_cut"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_min",		double_t, 0, "x_min",    	  -1.4, -10, 10);
gen.add("x_max",		double_t, 0, "x_max",    	  0.8, -10, 10);
gen.add("y_min",		double_t, 0, "y_min",    	  1.8, -10, 10);
gen.add("y_max",		double_t, 0, "y_max",    	  2.8, -10, 10);
gen.add("z_min",		double_t, 0, "z_min",    	  -1.0, -10, 10);
gen.add("z_max",		double_t, 0, "z_max",    	  0.4, -10, 10);

exit(gen.generate(PACKAGE, "laser_cut", "laser_cut"))
